sirloin-jvmlib-util/src/main/kotlin/com/sirloin/jvmlib/util/SemanticVersion.kt

```diff
@@ -0,0 +1,13 @@
+package com.sirloin.jvmlib.util
+
+data class SemanticVersion(
```

Comment:
KDoc 을 영문으로 추가해 주시면 좋겠습니다.

```
/**
 * A semantic version definition and implementation.
 * 
 * [More documentation](https://semver.org/)
 *
 * @since 2022-05-20
 */
```